src/python/automation/demo6_build_rupture_set.py

```diff
@@ -0,0 +1,10 @@
+from py4j.java_gateway import JavaGateway
+gateway = JavaGateway()
+app = gateway.entry_point
+
+builder = app.getBuilder()
+result = builder.setMaxFaultSections(100)\
```

Comment:
This is just the best. There's minimal fuss on each side. The last time I did remote method invocation (RMI) with Java you had to compile stubs and do a dance at full moon. And that was just for Java-to-Java communication. Admittedly, it was like 20 years ago.

Reply:
Yes, I've been elaborating the tests, and I can drill into the Java objects and pull out all sorts of interesting metrics.